sn07.yml

```
@@ -225,14 +230,17 @@
     - usegalaxy-eu.fix-stuck-handlers # Restart handlers to prevent several classes of issues
     - usegalaxy-eu.log-cleaner # do not retain journalctl logs, they are unnecessary/risky under GDPR
     - usegalaxy-eu.fix-ancient-ftp-data # Remove FTP data older than 3 months, create FTP user directories
+
+    # Some of our 'cleanups' also generate telegraf format so this goes at end.
+    # 25.11.2022: Moved here becuase the galaxy-procstat role fails
+    # as it is not able to find the /etc/telegraf/telegraf.d/ directory
+    - dj-wasabi.telegraf
     - usegalaxy-eu.galaxy-procstat # Some custom telegraf monitoring that's templated
     - usegalaxy-eu.fix-missing-api-keys # Workaround for IE users not have a key set.
     - usegalaxy-eu.fix-user-quotas # Automatically recalculate user quotas and attribute ELIXIR quota to ELIXIR AAI user on a regular basis
 
-    # Some of our 'cleanups' also generate telegraf format so this goes at end.
-    - dj-wasabi.telegraf
     # - dev-sec.os-hardening
-    - dev-sec.ssh-hardening
+    - ssh_hardening
```

```suggestion
    - ssh_hardening #dev-sec.hardening collection
```
to avoid confusion, maybe?